scaling_performance/scaling_cluster_metrics.adoc

```diff
@@ -39,51 +37,58 @@ xref:../admin_guide/manage_nodes.adoc#infrastructure-nodes[infrastructure
 nodes].
 * Use persistent storage when configuring metrics. Set
 `USE_PERSISTENT_STORAGE=true`.
-* Keep the `METRICS_RESOLUTION=30` parameter in {product-title} metrics
-deployments. Using a value lower than the default value of `30` for
-`METRICS_RESOLUTION` is not recommended. When using the Ansible metrics
-installation procedure, this is the `openshift_metrics_resolution` parameter.
-* Closely monitor {product-title} nodes with host metrics pods to detect early
-capacity shortages (CPU and memory) on the host system. These capacity shortages
-can cause problems for metrics pods.
-* In {product-title} version 3.7 testing, test cases up to 25,000 pods were
-monitored in a {product-title} cluster.
+* use glusterfs as storage on top of nvme drives link:https://github.com/openshift/openshift-ansible/tree/master/roles/openshift_storage_glusterfs[openshift_storage_glusterfs]
 
 [[scaling-performance-capacity-planning]]
 == Capacity Planning for Cluster Metrics
 
-In tests performed with 210 and 990 {product-title} nodes, where 10500 pods
-and 11000 pods were monitored respectively, the Cassandra database grew at the
+A various tests performed with for different scales, we have seen the the Prometheus database grew at the
 speed shown in the table below:
 
-.Cassandra Database storage requirements based on number of nodes/pods in the cluster
+.Prometheus Database storage requirements based on number of nodes/pods in the cluster
 [options="header"]
 |===
-|Number of Nodes |Number of Pods |Cassandra Storage growth speed |Cassandra storage growth per day |Cassandra storage growth per week
-
-|210
-|10500
-|500 MB per hour
-|15 GB
-|75 GB
-
-|990
-|11000
-|1 GB per hour
-|30 GB
-|210 GB
+|Number of Nodes |Number of Pods |Prometheus storage growth per day |Prometheus storage growth per 15d |RAM Space (per scale size) |Network (per tsdb chunk)
+
+|50
+|1800
+|232.8 Mb
+|3.4 Gb
+|6 Gib
+|16 Mib
+
+|100
+|3600
+|374 Mb
+|5.6 Gb
+|10 Gib
+|26 Mib
+
+|150
+|5400
+|528 Mb
+|7.9 Gb
+|12 Gib
+|36 Mib
+
+|200
+|7200
+|691.2 Mb
+|10.3 Gb
+|14 Gib
+|46 Mib
 |===
 
 In the above calculation, approximately 20 percent of the expected size was added as
 overhead to ensure that the storage requirements do not exceed calculated value.
 
-If the `METRICS_DURATION` and `METRICS_RESOLUTION` values are kept at the
-default (`7` days and `15` seconds respectively), it is safe to plan Cassandra
-storage size requirements for week, as in the values above.
+The above calculation was developed for defaults openshift cluster-monitoring-operator
```

Comment:
for the default {product-title} *cluster-monitoring-operator*. For higher scale, edit the `promehutes_persistent_volume_size` variable, which defaults to `50Gib`.